api/sheets/places_in.md

```diff
@@ -1,58 +1,38 @@
 ---
 layout: default
-title: Places within a Place
+title: Retrieving Child Nodes
 nav_order: 2
 parent: Google Sheets
 grand_parent: API
 ---
 
-# Retrieve places contained within another place
+# Retrieving Child Nodes
```

Comment:
Please update

Reply:
I changed it to "Retrieving Places Contained Within Another Place"